lib/rust/ensogl/core/src/animation/frp/animation.rs

```diff
@@ -56,14 +51,16 @@ where mix::Repr<T>: inertia::Value
     pub fn new(network: &frp::Network) -> Self {
         frp::extend! { network
             value_src <- any_mut::<T>();
+            on_end_src <- any_mut();
         }
         let on_step = Box::new(f!((t) value_src.emit(mix::from_space::<T>(t))));
-        let simulator = AnimationSimulator::<T>::new(on_step, (), ());
-        // FIXME[WD]: The precision should become default and should be increased in all simulators
+        let on_end = Box::new(f!((_) on_end_src.emit(())));
+        let simulator = AnimationSimulator::<T>::new(on_step, (), on_end);
+        // FIXME[WD]: The precision should should be increased in all simulators
         //            that work with pixels. The reason is that by default the simulator should
         //            give nice results for animations in the range of 0 .. 1, while it should not
         //            make too many steps when animating bigger values (like pixels).
```

Comment:
This FIXME should go now to the definitiuon side of DEFAULT_PRECISION. Otherwise it would be hard to spot it.